charms/tensorboard-controller/src/prometheus_alert_rules/TensorboardControllerDown.rule

```diff
@@ -0,0 +1,8 @@
+alert: TensorboardControllerDown
+expr: up == 0
+for: 5m
```

Comment:
Is this the recommended down time for production environments? 5m seems too much to me, but maybe you have more context.

Reply:
We discuss this with Nishant, and he agreed that it's good time. The reason is that if we make it short it will be triggered too much (e.g. during charm upgrade) and it's not possible to overwritten it with [cos-configuration](https://charmhub.io/cos-configuration-k8s), because you can overwrite it only with more restrictive configuration (e.g. now it's 5 and you can use 3).

Reply:
However, after discussion with @orfeas-k, we realized that we already have alert rules defined in [KF093](https://docs.google.com/document/d/15A5-F2ULGTGHhpFdP3PSqMBbHFnbLWugwbNF6uuG_1I/edit#heading=h.9qgo07lpznnt), and we should use them in all charms. So I'm going to update this one also.